code/modules/telesci/gps.dm

```diff
@@ -34,6 +34,8 @@ var/list/GPS_list = list()
 /obj/item/device/gps/AltClick(mob/user)
 	if(!in_range(src, user))
 		return
+	if(!user.canUseTopic(user))

```

Comment:
Okay so I tested this on my own test server.

Observers/ghosts cannot use the washing machine unless they are an adminGhost, did you overlook this when testing. I suspect this is what's causing the confusion?

Basically it doesn't seem to be broken and it's just the GPS needs a user.canUseTopic(src)